src/ceph-disk/ceph_disk/main.py

```diff
@@ -2385,6 +2385,7 @@ def create_key(self):
         command_check_call(
             [
                 'ceph',
+                '--name', 'client.bootstrap-osd',
```

Comment:
Oh yeah.. --keyring.  See the 
<pre>
        activate_key_template='{statedir}/bootstrap-osd/{cluster}.keyring',
</pre>
bits